java/code/src/com/redhat/rhn/frontend/xmlrpc/org/trusts/OrgTrustHandler.java

```diff
@@ -207,10 +207,10 @@
                 OrgManager.getSharedChannels(loggedInUser,
                         loggedInUser.getOrg(), trustOrg));
 
-        details.put("systems_migrated_to",
+        details.put("systems_transferred_to",
                 OrgManager.getMigratedSystems(loggedInUser,
                         trustOrg, loggedInUser.getOrg()));
-        details.put("systems_migrated_from",
+        details.put("systems_transferred_from",
```

Comment:
This break compatibility

Reply:
Changed it so that both the old and the new one are returned with the old one documented as deprecated.